Simplified/NYPLBookRegistry.m

```diff
@@ -469,10 +469,13 @@ - (void)updateAndRemoveBook:(NYPLBook *)book
     if(record) {
       self.identifiersToRecords[book.identifier] = [[record recordWithBook:book] recordWithState:NYPLBookStateUnregistered];
       [self broadcastChange];
-      // Queue this up so it happens after the broadcast is done.
-      [[NSOperationQueue mainQueue] addOperationWithBlock:^{
-        [self performSynchronizedWithoutBroadcasting:^{
-          [self removeBookForIdentifier:book.identifier];
+      // Timer delays removing from identifiersToRecords to give interfaces enough time for update.
```

Comment:
I think I misunderstood what `interfaces` meant here... reading your other comment below you're referring to views, correct?

Reply:
yes, right, = "to give views enough time"